File: internal/models/account.go
package models

type Account struct {
	FirstName string `json:"first_name" rethinkdb:"first_name" validate:"required"`
	LastName  string `json:"last_name" rethinkdb:"last_name" validate:"required"`
	Birthday  string `json:"birthday" rethinkdb:"birthday" validate:"required"`
	Password  string `json:"password,omitempty" rethinkdb:"password" validate:"required,min=6"`
	Username  string `json:"username" rethinkdb:"username" validate:"required,min=6"`
	UserID    string `json:"user_id" rethinkdb:"user_id" validate:"required"`
}

File: schemas/account.py
from marshmallow_sqlalchemy import ModelSchema
from models.friends import FriendModel
from models.user import UserModel
from models.cards import CardModel
from marshmallow import fields
from server.instance import db

class UserSchema(ModelSchema):
    class Meta(ModelSchema.Meta):
        model = UserModel
        sqla_session = db.session       
        user_id = fields.Integer(required=True)
        first_name = fields.String(required=True)
        last_name = fields.String(required=True)
        birthday = fields.String(required=True)
        username = fields.String(required=True)
        password = fields.String(required=True)

class CardSchema(ModelSchema):
    class Meta(ModelSchema.Meta):
        model = CardModel
        sqla_session = db.session
        user_id = fields.String(required=True)
        card_id = fields.Integer(required=True)
        title = fields.String(required=True)
        pan = fields.String(required=True)
        expiry_mm = fields.String(required=True)
        expiry_yyyy = fields.String(required=True)
        security_code = fields.String(required=True)
        date = fields.String(required=True)

class FriendSchema(ModelSchema):
    class Meta(ModelSchema.Meta):
        model = FriendModel
        sqla_session = db.session
        friend_id = fields.String(required=True)
        user_id = fields.String(required=True)
        username = fields.String(required=True)